src/newSeed/baseStage.scss

```diff
@@ -0,0 +1,223 @@
+@import "styles/colors";
+// This styles are common to all the stages components
+
+.page.stageContainer {
+  &.stage1 {
+    .infoContainer {
+      background-color: $color-black-2;
+      padding: 2em;
+      border-radius: 15px;
```

Comment:
em?

Reply:
Yes 😂 to resize with font size

Reply:
It allows for better responsiveness and i investigated that

Reply:
the comment is on a line that is using px

Reply:
Aha, for border-radius